examples/hardhat-sol/scripts/mint-token.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import ERC20_ABI from '../artifacts-zk/contracts/TestToken.sol/TestToken.json';
+// ANCHOR: start
+import { ethers } from 'ethers';
+
+// Address of the ERC20 token contract
+const TOKEN_CONTRACT_ADDRESS = '';
+// Wallet that will receive tokens
+const RECEIVER_WALLET = '';
+// Amount of tokens to mint in ETH format, e.g. 1.23
+const TOKEN_AMOUNT = '';
+
+(async () => {
+  try {
+    // ANCHOR_END: start
+    // // Note that the script needs the ABI which is generated from the compilation artifact.
+    // // Make sure contract is compiled and artifacts are generated
+    // const artifactsPath = `browser/contracts/artifacts/TestToken.json`
+    // console.log(artifactsPath)
+
+    // const metadata = JSON.parse(await remix.call('fileManager', 'getFile', artifactsPath))
+
+    // 'web3Provider' is a remix global variable object
+    // const signer = (new ethers.BrowserProvider(web3Provider)).getSigner(0)
```

Comment:
This commented out code intentional?

Reply:
I can remove, but it's just to make it clear that this file isn't an exact match to the template the user gets.